BrQPassword.vue

```diff
@@ -18,7 +18,7 @@
     <template v-slot:append>
       <q-icon
         :name="visible ? hideIcon : showIcon"
-        class="cursor-pointer"
+        class="cursor-pointer toggle-icon"
```

Comment:
Do we need a custom css class? Can this be achieved by leveraging quasar? Would `q-ml-xs` suffice?

Reply:
I tried that initially but I feel like 10px looks better than 8px. We can use q-ml-xs if you would prefer.